src/web/mjs/connectors/ReManga.mjs

```diff
@@ -0,0 +1,110 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ReManga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'remanga';
+        super.label = 'ReManga';
+        this.tags = [ 'manga', 'webtoon', 'russian' ];
+        this.url = 'https://remanga.org';
+        this.api = 'https://api.remanga.org';
+    }
+
+    async _getMangaFromURI(uri) {
+        uri.searchParams.delete('subpath');
+        const slug = uri.pathname.split('/').pop();
+
+        const request = new Request(new URL('/api/titles/' + slug, this.api), this.requestOptions);
+        const { content } = await this.fetchJSON(request, 3);
+
+        return new Manga(this, content.dir, content.en_name || content.another_name);
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+
+        // Of course, I can get the all manga, but it's over 20 000
+        for (let page = 0; page <= 100; page++) {
+            const mangas = await this._getMangasFromPage(page);
+            mangaList.push(...mangas);
+        }
+
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        const requestUrl = new URL('/api/search/catalog/', this.api);
+
+        requestUrl.searchParams.set('ordering', '-rating');
+        requestUrl.searchParams.set('count', '30');
+        requestUrl.searchParams.set('page', page || '1');
+
+        const request = new Request(requestUrl, this.requestOptions);
+        const { content } = await this.fetchJSON(request);
+
+        return content.map(item => {
+            return {
+                id: item.dir,
+                title: item.en_name || item.rus_name,
+            };
+        });
+    }
+
+    async _getChapters(manga) {
+        const request = new Request(new URL('/api/titles/' + manga.id, this.api), this.requestOptions);
+        const { content } = await this.fetchJSON(request, 3);
+
+        const branch = content.branches[0];
+        const pagesCount = content.count_chapters > 60 ? Math.trunc(content.count_chapters / 60) : 1;
```

Comment:
Is this behavior correct?
`content.count_chapters = 50` and `content.count_chapters = 100` will both result in `1`